packages/wrangler/src/pages/constants.ts

```diff
@@ -1,10 +1,12 @@
 import { version as wranglerVersion } from "../../package.json";
 
+const isWindows = process.platform === "win32";
+
 export const MAX_ASSET_COUNT = 20_000;
 export const MAX_ASSET_SIZE = 25 * 1024 * 1024;
 export const PAGES_CONFIG_CACHE_FILENAME = "pages.json";
 export const MAX_BUCKET_SIZE = 40 * 1024 * 1024;
-export const MAX_BUCKET_FILE_COUNT = 2000;
+export const MAX_BUCKET_FILE_COUNT = isWindows ? 1000 : 2000;
```

Comment:
Let's add a comment here to explain why Windows needs a lower limit, so that people don't try to change it in the future.